tests/omise/EventTest.php

```diff
@@ -0,0 +1,36 @@
+<?php require_once dirname(__FILE__).'/TestConfig.php';
+
+class EventTest extends TestConfig {
+  /**
+   * OmiseEvent class must be contain methods as below.
+   *
+   */
+  public function testMethodExists() {
+    $this->assertTrue(method_exists('OmiseEvent', 'retrieve'));
+    $this->assertTrue(method_exists('OmiseEvent', 'reload'));
+    $this->assertTrue(method_exists('OmiseEvent', 'getUrl'));
+  }
+
+  /**
+   * Assert that a list of event object could be successfully retrieved.
+   *
+   */
+  public function testRetrieveEventListObject() {
+    $event = OmiseEvent::retrieve();
```

Comment:
@guzzilar how can you skip this line from authentication?, I have tried to do like this but I'm stuck in the authentication step.